Octokit/Http/HttpClientAdapter.cs

```diff
@@ -76,8 +76,8 @@ public HttpClientAdapter(IWebProxy webProxy)
                 {
                     contentType = GetContentMediaType(responseMessage.Content);
 
-                    // We added support for downloading images. Let's constrain this appropriately.
-                    if (contentType == null || !contentType.StartsWith("image/"))
+                    // We added support for downloading images and zip-files. Let's constrain this appropriately.
+                    if (contentType == null || (!contentType.StartsWith("image/") && !contentType.StartsWith("application/")))
```

Comment:
Just a heads-up - this change also traps `application/json` output which breaks serialization (I should have caught this). I've updated it for the latest release to be more strict: https://github.com/octokit/octokit.net/blob/master/Octokit/Http/HttpClientAdapter.cs#L86